paper.md

```diff
@@ -68,7 +68,7 @@ To support and disseminate the use of our integration work, we also planned to c
 
 Writing a data protection impact assessment for a scientific project is a challenging task. Input from multiple key groups is required to obtain complete and precise information. In addition to data stewards, which are commonly the ones in charge of the information collection since their knowledge overlaps all other involved domains, the process involves researchers themselves bringing knowledge of about the actual data and related scientific processes, data managers and IT staff describing the local IT infrastructures and external collaborators reporting on the shared data processing tasks. Finally, legal representatives (e.g. data protection officer) validates the resulting document with existing contracts, institutional policies and approves the final version.
 
-DPIA is commonly written using spreadsheet or text based templates with questions and predefined answers. This approach suffers from the absence of branching logic, overwhelming and poorly structured content and lack of support for collaborative editing. Furthermore, subsequent updates of the template make the whole process laborious and unnecessarily complex. Dedicated tools (e.g. PIA [citation PIA]) are also available and they do bring substantial help but they in general lack the ability of full customization of the template.
+DPIA is commonly written using spreadsheet or text based templates with questions and predefined answers. This approach suffers from the absence of branching logic, overwhelming and poorly structured content and lack of support for collaborative editing. Furthermore, subsequent updates of the template make the whole process laborious and unnecessarily complex. Dedicated tools (e.g. [Processing Impact Assessment](https://www.cnil.fr/en/open-source-pia-software-helps-carry-out-data-protection-impact-assesment) from Commission Nationale de l'Informatique et des Libertés (CNIL)) are also available and they do bring substantial help but they in general lack the ability of full customization of the template.
```

Comment:
This approach suffers from the absence of **a** branching logic, overwhelming and poorly structured content, and **a** lack of support for collaborative editing.

Reply:
[Processing Impact Assessment](https://www.cnil.fr/en/open-source-pia-software-helps-carry-out-data-protection-impact-assesment) from **the** Commission Nationale de l'Informatique et des Libertés (CNIL)) are also available and they do bring substantial help but  in general they lack the ability of a full customization of the template.

Reply:
**A** DPIA is commonly written using spreadsheet